<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Operaciones">
      <widget class="QWidget" name="ListaSimple">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>281</width>
             <height>241</height>
            </rect>
           </property>
           <property name="title">
            <string>Insertar</string>
           </property>
           <widget class="QLabel" name="PosicionLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>63</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Posicion</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="editPosLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>113</width>
              <height>28</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="NumeroLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>110</y>
              <width>63</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Numero</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="editNumLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>140</y>
              <width>113</width>
              <height>28</height>
             </rect>
            </property>
           </widget>
           <widget class="QPushButton" name="btnInsertarLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>200</y>
              <width>83</width>
              <height>29</height>
             </rect>
            </property>
            <property name="text">
             <string>Insertar</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="gbBorrarLS">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>270</y>
             <width>281</width>
             <height>251</height>
            </rect>
           </property>
           <property name="title">
            <string>Eliminar</string>
           </property>
           <widget class="QLabel" name="PosicionLS_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>40</y>
              <width>63</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Posicion</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="editDelPosLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>70</y>
              <width>113</width>
              <height>28</height>
             </rect>
            </property>
            <property name="placeholderText">
             <string>0, 1, 2.....</string>
            </property>
           </widget>
           <widget class="QPushButton" name="btnBorrarLS">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>120</y>
              <width>83</width>
              <height>29</height>
             </rect>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
